beta_gouv_datasets.yml

```diff
@@ -0,0 +1,10 @@
+datasets:
+  - id: metromobilite
+    name: Grenoble
+    gtfs: https://www.metromobilite.fr/data/Horaires/SEM-GTFS.zip
+    gtfs-rt: https://data.metromobilite.fr/api/gtfs-rt/GAM/trip-update
+
+  - id: poitiers
+    name: Poitiers
+    gtfs: https://data.grandpoitiers.fr/api/v2/catalog/datasets/mobilite-arret-services-theoriques-bus-hiver/files/400d7da94eaacb5e52c612f8ac28e420
+    gtfs-rt: https://app.mecatran.com/utw/ws/gtfsfeed/realtime/poitiers?apiKey=5f60015b5c16355371622b3c70770c6736384c02
```

Comment:
If we want it to be reusable by other people we have to think about a way to have a proper configuration file.

Reply:
Yep, I don't like to have conf file in this repo.
Maybe we could put in cellar in clevercloud, or in another repo.